rites/roman1969/src/models/liturgical-day-def.ts

```diff
@@ -342,8 +342,7 @@ export class LiturgicalDayDef implements BaseLiturgicalDayDef {
     if (input.titles) {
       martyrology.forEach((m, i) => {
         // TODO: refactor this to avoid non-null assertion
```

Comment:
:pick: Could we call this `TODO` comment done if you added `if (input.titles)`? If yes, we should remove it.

Reply:
sure, can do